LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ModuleConnectionHost.cs

```diff
@@ -26,9 +26,12 @@ public sealed class ModuleConnectionHost : IAsyncDisposable
         private readonly LoRaDeviceAPIServiceBase loRaDeviceAPIService;
         private readonly ILogger<ModuleConnectionHost> logger;
         private readonly Counter<int> unhandledExceptionCount;
+        private readonly Counter<int> forceClosedConnections;
         private ILoraModuleClient loRaModuleClient;
         private readonly ILoRaModuleClientFactory loRaModuleClientFactory;
 
+        public const string DroppedConnectionLog = "Device connection was dropped ";
```

Comment:
nit: I would prefer `closed` vs. `dropped` - dropped sounds unexpected.

Reply:
Done in d489a749d8a517f62eb764a7522091ec36527b1c